packages/api/src/api/contributions.ts

```diff
@@ -0,0 +1,233 @@
+import express, { Router } from 'express';
+import { env } from '../env';
+import { Contribution } from '../entities/Contribution';
+import { User } from '../entities/User';
+import logger from '../logger';
+import { Project } from '../entities/Project';
+
+export const contributions = Router();
+contributions.use(express.json());
```

Comment:
[This is already being done for the main app](https://github.com/AmericanAirlines/Project-X/blob/bce50078450b9e2efe1c5cf71e601f7c923c6ef7/packages/api/src/index.ts#L17) so you shouldn't need it here as well
```suggestion
```

Reply:
removed